src/plugins/autoColumnSize/autoColumnSize.js

```diff
@@ -342,7 +349,7 @@ class AutoColumnSize extends BasePlugin {
     let width = defaultWidth;
 
     if (width === void 0) {
-      width = this.widths[column];
+      width = this.columnWidthsMap.getValueAtIndex(column);
```

Comment:
I'm not sure if using `getValueAtIndex`is proper. The `getColumnWidth` is used as callback for `modifyColWidth` hook listener, but [modifyColWidth](https://handsontable.com/docs/7.1.0/Hooks.html#event:modifyColWidth) gives visual index. Probably it should be translated to the physical index. Probably the bug has existed so far.

Reply:
You're right. I recognised a similar change as a potential bug when I worked on #6173.